ballerina/tests/14-error-test.bal

```diff
@@ -273,3 +273,36 @@ function TestDuplicateKey() returns error? {
     test:assertTrue(strings:includes(sqlerror.message(), "violated"),
                 sqlerror.message());
 }
+
+@test:Config {
+    groups: ["error"]
+}
+function TestUnderflowException() returns error? {
+    Client dbClient = check new (HOST, USER, PASSWORD, DATABASE, PORT);
+    _ = check dbClient->execute(`CREATE TABLE test( VALUE DOUBLE PRECISION)`);
+    decimal value = 1e-234;
+    sql:ParameterizedQuery insertQuery = `Insert into test (VALUE) values (${value})`;
+    sql:ExecutionResult|error insertResult = dbClient->execute(insertQuery);
+    insertResult = dbClient->execute(insertQuery);
+    check dbClient.close();
+    sql:DatabaseError sqlerror = <sql:DatabaseError>insertResult;
+    test:assertTrue(strings:includes(sqlerror.message(), "Underflow Exception trying to bind 1E-234"),
```

Comment:
Shouldn't we catch this exception and give a proper error message. Seems like these are thrown when the decimal min value is exceeded, isn't it?

Reply:
The existing error message is:  
`Error while executing SQL query: Insert into test (VALUE) values ( ? ). Internal Error: Underflow Exception trying to bind 1E-234` 
We can catch this error and improve an error message, but this error is produced when executing the query. Hence, we can't extract the particular value which has this issue. So we can't add that value in the error message. This message will also have a lack of information. Therefore, I think it's good to have this message.

Reply:
Ack